indra-common/src/main/kotlin/net/kyori/indra/IndraExtension.kt

```diff
@@ -117,6 +117,13 @@ open class IndraExtension @Inject constructor(objects: ObjectFactory) {
     url = "https://opensource.org/licenses/Apache-2.0"
   ))
 
+  fun gpl30OnlyLicense() = this.license.set(License(
+    spdx = "GPL-3.0-only",
+    name = "GNU General Public License version 3",
```

Comment:
SPDX adds " only" to the end of the name, but I don't think we should?

Reply:
yeah, I agree